test/Microsoft.AspNetCore.Razor.Test/CodeGenerators/CSharpCodeGeneratorTest.cs

```diff
@@ -55,13 +65,123 @@ public void ChunkTreeWithUsings()
             if (!string.Equals(expectedOutput, result.Code, StringComparison.Ordinal))
             {
                 BaselineWriter.WriteBaseline(
-                    @"test\Microsoft.AspNetCore.Razor.Test\TestFiles\CodeGenerator\Output\CSharpCodeGenerator.cs",
+                    Path.Combine(_testOutputDirectory, "Output", "CSharpCodeGenerator.cs"),
```

Comment:
Also looks like `"test"` and `"Microsoft.AspNetCore.Razor.Test"` are missing from the start of the `params` array and `"Output"` is already in `_testOutputDirectory`.

Probably best to enable baseline generation (temporary edit of the `project.json` file) and make sure things are written where they're expected.